sciserver_catalogs/environment.yml

```diff
@@ -35,10 +34,12 @@ dependencies:
   - nc-time-axis
   - fastjmd95
   - cf_xarray
-  - oceanspy==0.2.0
   - pip
+  - pooch 
```

Comment:
```suggestion
  - pooch
  - h5netcdf
  - scipy
  - pydap
  - zarr
  - fsspec
  - cftime
  - rasterio
  - cfgrib
  - matplotlib
  - cf_xarray
  - ipykernel
  - numba
  - xoak
  - scikit-learn
  - pys2index
  - cmasher
  - imageio
  - pre-commit
```

Reply:
I like installing all these explicitly this too. I think the only reason we didn't define them explicitly is because @asiddi24 noticed that I bunch of packages (like `zarr` and `fsspec`) get installed anyway as other packages get installed. Either way, I also preferr to defined them explicitly in the environment.